### PR TITLE
Added wall button on home page

### PR DESCRIPTION
**Pull request by satcasm**
Fixes: #812 

Screenshot:
image

### CODE REVIEW

**Comment by Pranav016** [comment]
How does it look in mobile view ?

File: public/css/styles.css [hunk 1]
```diff
@@ -437,6 +437,26 @@ input[name='perPage'] {
...
+	color: white;
+	transition: 0.2s;
+	border: 3px solid white;
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.test-wall-btn:hover {
+	background: #129fb4;
+}
+
```
Comment (Pranav016) [inline public/css/styles.css:459]: you don't need to add all the styles again, just reuse the `button` and `arrow` class for adding the styles.
Add this class only for positioning the button.

**Review by Pranav016 — CHANGES_REQUESTED** [review]

**Comment by satcasm** [comment]
> How does it look in mobile view ?

The responsiveness of the site is pretty messed up before ```900pxs```.
blog1

**Comment by Pranav016** [comment]
> > How does it look in mobile view ?
> 
> The responsiveness of the site is pretty messed up before `900pxs`.
> blog1

I checked on my system, the website looks fine in mobile view, the issue is only for tablet view.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
I saw the demo video and I think we shouldn't have the testimonial wall button at the top in mobile view. I think we should put it below the `hello there` h1 so that in mobile view it comes under that heading.


I would like to hear your suggestions as well regarding the positioning of the button

**Comment by satcasm** [comment]
Let me try with that look and will let you know how that looks.

**Comment by satcasm** [comment]
So @Pranav016 , should I go with this or not?

**Comment by Pranav016** [comment]
Yes for now we can do this, maybe in the future I'll come up with a better position for the button. You can push your code now 👍

**Comment by satcasm** [comment]
@Pranav016 pushed the changes have a look.
Thanks!

**Review by Pranav016 — APPROVED** [review]

**Comment by RushanKhan1** [comment]
@all-contributors please add @satcasm for code

**Pull request merged**